packages/webapp/pages/account/notifications.tsx

```diff
@@ -40,10 +49,21 @@ const AccountNotificationsPage = (): ReactElement => {
   const { user } = useContext(AuthContext);
   const {
     personalizedDigest,
-    isLoading: isPersonalizedDigestLoading,
+    readingReminder,
     subscribePersonalizedDigest,
     unsubscribePersonalizedDigest,
   } = usePersonalizedDigest();
+  const [digestTimeIndex, setDigestTimeIndex] = useState(8);
+  if (personalizedDigest?.preferredHour !== digestTimeIndex) {
+    setDigestTimeIndex(personalizedDigest?.preferredHour);
+  }
+  const [readingTimeIndex, setReadingTimeIndex] = useState(
```

Comment:
I do think declaring all state on top then doing manipulations to it is better when u read the code.